rpc/core/src/notify/message.rs

```diff
@@ -4,7 +4,7 @@ use std::sync::Arc;
 
 #[derive(Clone, Debug)]
 pub(crate) enum DispatchMessage {
-    Send(Arc<Notification>),
+    Send(Box<Notification>),
```

Comment:
Why do we need a box?

Reply:
Done. this was to patch clippy warning away, but replaced with ignore and TODO.